static/app/views/performance/landing/content.tsx

```diff
@@ -1,337 +0,0 @@
-import {Component, Fragment} from 'react';
```

Comment:
I've deleted this file since it seemed that it wasn't used anywhere anymore. Let me know if this is a breaking change 🙏